README.md

```diff
@@ -33,13 +33,31 @@ yarn
 Define a `.env` file and define, at minimum, the following (see [here](./src/lib/env.ts) for more settings):
 
 ```
-OVAL_ADDRESS=0x420 // The address of the Oval contract you want to target with the Oval Node.
-SENDER_PRIVATE_KEY=0x6969 // The private key of the permissioned actor who can call unlockLatestValue on the associated Oval.
-AUTH_PRIVATE_KEY=0x6666 // The root private key used to derive searcher specific keys for signing bundles.
-REFUND_ADDRESS=0x42069 // The refund address you want to send the OEV kickback to.
-PROVIDER_URL=https://mainnet.infura.io/v3/abcdef // Ethereum mainnet RPC provider.
+SENDER_PRIVATE_KEY=<your_sender_private_key>  # Private key of the actor authorized to call unlockLatestValue on the Oval.
```

Comment:
If using `OVAL_CONFIGS` then there is no need for `OVAL_ADDRESS`, `SENDER_PRIVATE_KEY`, `REFUND_ADDRESS` and `REFUND_PERCENT` - these are here only for backwards compatibility when serving one Oval contract instance. So either we don't document any of them or document all 4 explaining you don't need `OVAL_CONFIGS` then.